### PR TITLE
Add a gym wrapper for pysim envs

### PR DESCRIPTION
**Pull request by miterion**
Allows the usage of pysim envs with agents expecting a gym environment.

### CODE REVIEW

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        return self._wrapped_env.reset()
+
+    def render(self, mode="human"):
+        self._wrapped_env.render(mode=RenderMode(text=False, video=True, render=False), render_step=1)
+
+    def close(self):
+        self._wrapped_env.close()
+
+    def seed(self, seed=None):
+        pyrado.set_seed(seed)
```
Comment (famura) [inline Pyrado/pyrado/environments/pysim/gym-wrapper.py:71]: Add documentation to the functions. You can copy them from the Env or SimPyEnv.

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    @staticmethod
+    def conv_space_from_simulacra(space: Space) -> gym.spaces.Space:
```
Comment (famura) [inline Pyrado/pyrado/environments/pysim/gym-wrapper.py:74]: Please rename "simurlacra" to "pyrado"

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    """
+
+    metadata = {"render.modes": ["human"]}
+
+    def __init__(self, env: SimPyEnv):
+        self._wrapped_env = env
+
+        # translate environment parameters
+        self.action_space = self.conv_space_from_simulacra(self._wrapped_env.act_space)
+        self.observation_space = self.conv_space_from_simulacra(self._wrapped_env.obs_space)
```
Comment (famura) [inline Pyrado/pyrado/environments/pysim/gym-wrapper.py:56]: Please add a constructor like for the other env classes.

Okease Capitalize "translate"

File: README.md [hunk 4]
```diff
@@ -74,7 +74,10 @@ conda activate pyrado
 conda install -y blas cmake lapack libgcc-ng mkl mkl-include patchelf pip pycairo setuptools -c conda-forge
 pip install -r requirements.txt
 ```
-
+If you want to use the gym wrapper
+```
+pip install gym
+```
```
Comment (famura) [inline README.md:80]: I understand that you didn't add it to the requirements. Do you see any change we can test the wrapper using pytest?

**Review by famura — CHANGES_REQUESTED** [review]

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py (hunk 2 above)
Comment (miterion) [reply Pyrado/pyrado/environments/pysim/gym-wrapper.py:74]: Done in 6937270512491a2acf4de4b2d99e2de3b969f382

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py (hunk 1 above)
Comment (miterion) [reply Pyrado/pyrado/environments/pysim/gym-wrapper.py:71]: Done in 6937270512491a2acf4de4b2d99e2de3b969f382

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py (hunk 2 above)
Comment (miterion) [reply Pyrado/pyrado/environments/pysim/gym-wrapper.py:74]: Done in c135c6462c430522aeb231d372d783ab0b856808

File: Pyrado/pyrado/environments/pysim/gym-wrapper.py (hunk 3 above)
Comment (miterion) [reply Pyrado/pyrado/environments/pysim/gym-wrapper.py:56]: Capitalized in 7c27e1d4ff

File: README.md (hunk 4 above)
Comment (miterion) [reply README.md:80]: Well, there is the option of installing stable-baselines3 and use their [gym environment checker](https://stable-baselines3.readthedocs.io/en/master/common/env_checker.html). Other than that I think it is a bit difficult.

Comment (famura) [reply README.md:80]: I defintely dont want to install `stable-baselines` to test this wrapper. What I meant is to write pytests with a marker which checks if gym is importable. Just like for MuJoCo...

**Review by famura — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (miterion) [reply README.md:80]: Done in ac97a55

**Review by famura — DISMISSED** [review]

**Comment by famura** [comment]
Thank you very much @miterion for your great contribution.

**Review by famura — APPROVED** [review]

**Pull request merged**